modal/image.py

```diff
@@ -1299,6 +1299,140 @@ def build_dockerfile(version: ImageBuilderVersion) -> DockerfileSpec:
             gpu_config=parse_gpu_config(gpu),
         )
 
+    def uv_sync(
+        self,
+        uv_project_dir: str = "./",  # Path to local uv mananged project
+        *,
+        force_build: bool = False,  # Ignore cached builds, similar to 'docker build --no-cache'
+        group: Optional[str] = None,  # Dependency group to install using `uv sync --group`
```

Comment:
Should this also accept a list? Or does `uv sync` only allow at most one `--group` at a time?